### PR TITLE
Upgrade the version to 3.0.0

### PR DESCRIPTION
**Pull request by BadSingleton**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+* Added bindings for FbxAnimCurveKey methods to set and get tangent mode and data
+* Added bindings for FbxAxisSystem::DeepConvertScene
+
+BUGFIXES
+* The FBX SDK C# Bindings package now supports the IL2CPP backend.
+
+KNOW ISSUES
+* The FBX SDK C# Bindings package cannot the used on Linux CentOS 7
+
 ## [2.0.0-preview.3] - 2018.01-03
```
Comment (benoithudson) [inline CHANGELOG.md]: This date is weird; the format is wrong, and the year seems wrong (or else we're time travelling, shipping preview.3 months before preview.2). Can you drive-by check when we actually released it and fix it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+* Upgraded to FBX SDK 2020.0
+* Added bindings for FbxAnimCurve::KeySetTangents and FbxAnimCurve::KeyGetTangents
+* Added bindings for FbxAnimCurveKey methods to set and get tangent mode and data
+* Added bindings for FbxAxisSystem::DeepConvertScene
+
+BUGFIXES
+* The FBX SDK C# Bindings package now supports the IL2CPP backend.
+
+KNOW ISSUES
+* The FBX SDK C# Bindings package cannot the used on Linux CentOS 7
```
Comment (benoithudson) [inline CHANGELOG.md]: "on Linux is not compatible with CentOS 7, use Ubuntu [version???] instead."

File: CHANGELOG.md (hunk 1 above)
Comment (BadSingleton) [reply CHANGELOG.md]: Git blame says 2018-12-03

Comment (BadSingleton) [reply CHANGELOG.md]: I believe this is a typo
